cortx-motr.spec.in

```diff
@@ -201,10 +219,14 @@ CORTX Motr IVT.
 %setup -q
 
 %build
-%configure %{with_linux} %{with_lustre} %{configure_opts}
-%if !%{with ivt}
-make %{?_smp_mflags}
+%if  "%{CONFIG_ARCH_TYPE}"=="x86_64"
+#x86_64 platform
```

Comment:
Drop this comment. It's obvious from `if` above that it's about x86_64 platform.